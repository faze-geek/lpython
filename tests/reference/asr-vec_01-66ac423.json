{
    "basename": "asr-vec_01-66ac423",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/vec_01.py",
    "infile_hash": "eb012561316a4c1f1359bf7b5a0b9aee40944ece197a5bafd387a19f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-vec_01-66ac423.stderr",
    "stderr_hash": "06e261dce693fc9366339d10eb1f41346f62eb47cee11c994d0b3d08",
    "returncode": 2
}