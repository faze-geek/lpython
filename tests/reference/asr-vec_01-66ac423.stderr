semantic error: 'f64[9217]' object does not support item assignment
  --> tests/../integration_tests/vec_01.py:10:9
   |
10 |         b[i] = 5.0
   |         ^^^^^^^^^^ 
